Source/Utils/WarpXConst.H

```diff
@@ -20,6 +23,10 @@ namespace MathConst
 
 // Physical constants. Values are the 2018 CODATA recommended values
 // https://physics.nist.gov/cuu/Constants/index.html
+//
+// New additions here should also be considered for addition to
+// `warpx_constants` in WarpXUtil.cpp, so that they're available in parsing and
+// evaluation of PICMI expressions, as well as the corresponding Python definitions
```

Comment:
```suggestion
// `warpx_constants` in WarpXUtil.cpp, so that they're available in parsing,
// and in `pywarpx/picmi.py::constants`, so that they're available in Python.
```